macro/gphysi.dat

```diff
@@ -0,0 +1,23 @@
+1***************************************************************************************************
+ *                                                                                                 *
+ *    G E A N T  Version 3.2111      DATE/TIME 180427/1612          R U N      1                   *
+ *                                                                                                 *
+ *      Data structure   Date   Time    GVERSN    ZVERSN                                           *
+ *                                                                                                 *
+ *           INIT       180427  1612    3.2111     3.77                                            *
+ *           KINE       180427  1612    3.2111     3.77                                            *
+ *           HITS       180427  1612    3.2111     3.77                                            *
+ *           DIGI       180427  1612    3.2111     3.77                                            *
+ *                                                                                                 *
+ *                                                                                                 *
+ *              Standard TPAR for this run are                                                     *
+ *                                                                                                 *
+ *  CUTGAM=  1.00 MeV  CUTELE=  1.00 MeV  CUTNEU=  1.00 MeV  CUTHAD=  1.00 MeV  CUTMUO=  1.00 MeV  *
+ *  BCUTE =  1.00 MeV  BCUTM =  1.00 MeV  DCUTE = 10.00 TeV  DCUTM = 10.00 TeV  PPCUTM=  2.04 MeV  *
+ *  IPAIR=  1.  ICOMP=  1.  IPHOT=  1.  IPFIS=  0.  IDRAY=  0.  IANNI=  1.  IBREM=  1.  IHADR=  1. *
+ *  IMUNU=  1.  IDCAY=  1.  ILOSS=  2.  IMULS=  1.  IRAYL=  1.  ILABS=  1.  ISYNC=  0.  ISTRA=  0. *
+ *                                                                                                 *
+ *                                                                                                 *
```

Comment:
What is this file supposed to do? (We have likely a different mechanism to set cuts and processes)

Reply:
(Maybe it was committed by error?)

Reply:
sure, will be removed

Reply:
It is GEANT3 output, I have no idea where it comes from, is it not exist here 
https://github.com/AllaMaevskaya/AliceO2/tree/fit-dev/Detectors/FIT/

Reply:
It is here:
https://github.com/AllaMaevskaya/AliceO2/tree/fit-dev/macro

Reply:
it was pushed by mistake, sure